src/Compiler/pars.fsy

```diff
@@ -985,8 +985,11 @@ classMemberSpfn:
                match optLiteralValue with
                | None -> m
                | Some e -> unionRanges m e.Range
-
+        
        let flags, leadingKeyword = $3
+       if leadingKeywordIsAbstract leadingKeyword && (Option.isSome $5 || Option.isSome getterAccess || Option.isSome setterAccess)
+       then errorR(Error(FSComp.SR.parsAccessibilityModsIllegalForAbstract(), mWhole))
```

Comment:
instead  of `mWhole` . can we use only the access modifier range ? e.g. `private|internal`

Reply:
fixed